Hold test Run job untill the xml reports ready for upgrade end-to-end

jyejare opened the pull request:
Fixes Polarion Test Run job failing with failed to find xml artifact from upgraded end-to-end tests job.

san7ket commented:
Looks like issue is with sauce plugin teardown, I think we can safely disable it for end2end as no UI tests are present

san7ket commented:
Now looks like the issue was with creating a artifacts >200 mbs took time, checking if removing unnecessary artifacts should fix it

jyejare commented:
@ntkathole, Changed PR according to @sjagtap comment and removed archiving unused/heavyweight files such as foreman_debug was 200mb almost and hence to archive the artifacts, the job was taking some time.
@san7ket, it's fixed now. Thanks.

Test Run: automation-upgraded-6.5-end-to-end-rhel7/27 triggers polarion-upgraded-test-run-6.5-rhel7/31.

ntkathole commented:
@jyejare do we need this PR then?

ntkathole commented:
@jyejare Thanks
